### PR TITLE
Aaron's bugs fixed

### LINKED ISSUE #287: `h` flag not implemented
The `h` flag just prints “Help Message”, and does nothing on the online interpreter.

[Try it Online!](https://vyxal.pythonanywhere.com/#WyJoIiwiIiwiIiwiIiwiIl0=)

### LINKED ISSUE #289: `?` cycles through the first two inputs
Instead of cycling through each input one by one, the `?` command only cycles through the first two inputs.

Example: [Try it Online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCI/Pz8/Pz8/Pz8/VyIsIiIsIjFcbjJcbjNcbjRcbjVcbjZcbjdcbjhcbjlcbjEwIl0=)

### LINKED ISSUE #292: Empty variable names error
Instead of using the nameless variable (`VAR_`), trying to use a variable without specifying a name just errors.

Example: [Try it Online!](https://vyxal.pythonanywhere.com/#WyJjIiwiIiwiMSAw4oaSWzMsIiwiIiwiIl0=)

### LINKED ISSUE #293: `a` flag casts inputs to strings
When wrapping inputs in a list, the `a` flag also casts those inputs to a string, even if it is a number or something similar.

Example: [Try it Online!](https://vyxal.pythonanywhere.com/#WyJhIiwiIiwi4oiRIiwiIiwiMVxuMlxuM1xuNFxuNVxuNiJd)

### PR DESCRIPTION
**Pull request by Lyxal**
Closes #287
Closes #289
Closes #292 
Closes #293 

Also fixes a small bug with recursion in lambdas where it'd return a list instead of just the value.

### CODE REVIEW

**Review by AMiller42 — APPROVED** [review]
lyxal handing out bug fixes like they’re cheap cigars

File: static/main.js [hunk 1]
```diff
@@ -171,8 +171,10 @@ function replaceHTMLChar(char) {
 }
 
 function copyToClipboard(arg) {
-    var el = document.getElementById(arg).value
-    navigator.clipboard.writeText(el)
+    var el = document.getElementById(arg)
```
Comment (chunkybanana) [inline static/main.js:174]: I wrote this precisely because `document.execCommand` is deprecated.

**Review by chunkybanana — APPROVED** [review]

File: static/main.js (hunk 1 above)
Comment (Lyxal) [reply static/main.js:174]: Mobile phone compatibility go brrrrr

Comment (chunkybanana) [reply static/main.js:174]: Oh :p. Well, let's leave it for now then :p

**Pull request merged**